pkg/core/config/tenant.go

```diff
@@ -132,5 +132,27 @@ func SetTenantConfig(i interface{}, t TenantConfiguration) {
 func NewTenantConfigurationFromEnv(_ *http.Request) (TenantConfiguration, error) {
 	c := TenantConfiguration{}
 	err := envconfig.Process("", &c)
+	c.SSOConfigs = appendSSOConfigs()
```

Comment:
@carmenlau I add a function which reads env SSO provider configs to tenant config, please help review. Thanks.

Reply:
Looks good:)